### PR TITLE
add missing tokens, fix/update docs

### PR DESCRIPTION
**Pull request by jgerlach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ npm install mx-design-tokens
...
+import { light, dark, targets } from 'mx-design-tokens'
 ```
 
 To build a custom colored themed object:
 
 ```
 import { buildTheme } from 'mx-design-tokens'
 
-const theme = buildTheme('light', { Primary300: '#876543' })
+const theme = buildTheme('light', targets.REACT, { Primary300: '#876543' })
```
Comment (phantomxc) [inline README.md:24]: Thanks for updating this.

File: docs/tokens.md [hunk 2]
```diff
@@ -279,3 +279,289 @@
...
+| ButtonTransparent | #2F73DA | ![#2F73DA](https://placehold.it/15/2F73DA/000000?text=+) |
+| ButtonTransparentHover | #165ECC | ![#165ECC](https://placehold.it/15/165ECC/000000?text=+) |
+| ButtonTransparentDisabled | #6A7381 | ![#6A7381](https://placehold.it/15/6A7381/000000?text=+) |
+| ButtonDestructive | #FFFFFF | ![#FFFFFF](https://placehold.it/15/FFFFFF/000000?text=+) |
+| ButtonDestructiveDisabled | #A8B1BD | ![#A8B1BD](https://placehold.it/15/A8B1BD/000000?text=+) |
+| ButtonTransparentTertiary | #49505A | ![#49505A](https://placehold.it/15/49505A/000000?text=+) |
+| ButtonLinkTertiary | #49505A | ![#49505A](https://placehold.it/15/49505A/000000?text=+) |
+| TabActive | #2F73DA | ![#2F73DA](https://placehold.it/15/2F73DA/000000?text=+) |
+| LeftNavItemActive | #165ECC | ![#165ECC](https://placehold.it/15/165ECC/000000?text=+) |
+| Tooltip | #FFFFFF | ![#FFFFFF](https://placehold.it/15/FFFFFF/000000?text=+) |
```
Comment (phantomxc) [inline docs/tokens.md]: Can you actually update the script to delete the doc file first? It just appends to that file.

File: src/tokens/boxShadow.js [hunk 3]
```diff
@@ -11,12 +16,12 @@ const light = core => ({
...
-  DropdownMenu: "none",
-  Modal: "none",
-  BottomBar: "none",
-  Drawer: "none",
+  Card: 'none',
+  Container: 'none',
+  DropdownMenu: 'none',
+  Modal: 'none',
+  BottomBar: 'none',
+  Drawer: 'none',
```
Comment (phantomxc) [inline src/tokens/boxShadow.js]: I think null or an empty string would be safer cross environment.

File: docs/tokens.md (hunk 2 above)
Comment (jgerlach) [reply docs/tokens.md]: I just pushed a change that I believe works. I'm not a node expert, but I did google it.

**Comment by phantomxc** [comment]
:shipit:

**Pull request merged**